app/front/javascripts/shared/controllers/siret_autocomplete_controller.js

```diff
@@ -32,35 +48,35 @@ export default class extends Controller {
     })
   }
 
-  connect() {
-    if (exists(this.fieldTarget.dataset.defaultValue)) {
-      document.querySelector('#solicitation-siret').value = this.fieldTarget.dataset.defaultValue
-    }
-  }
-
   manageSourceError(results) {
-    console.warn(results.error)
+    this.loaderTarget.style.display = 'none'
+    this.statusMessage = results.error
   }
 
-  manageSourceSuccess() {
-    // here, do nothing. Check children
+  manageSourceSuccess(items) {
+    this.loaderTarget.style.display = 'none'
+    this.statusMessage = (items.length == 0) ? "Aucune entreprise trouvée" : null
   }
 
-  onConfirm() {
-    // here, do nothing. Check children
+  onConfirm(option) {
+    if (option) {
+      this.fillSiretField(option);
+    }
   }
 
   // Récupération des résultats ----------------------------------------------------
 
   async fetchEtablissements(query) {
-    let params = `query=${query}&non_diffusables=${this.displayNonDiffusableSiret()}`;
-    let response = await fetch(`/rech-etablissement.json?${params}`, {
+    this.loaderTarget.style.display = 'block'
+    let baseUrl = this.fieldTarget.dataset.url
+    let params = `query=${query}`;
+    let response = await fetch(`${baseUrl}.json?${params}`, {
       credentials: "same-origin",
     });
     // Au cas où autre chose que du json est renvoyé
     try {
       let data = await response.json();
-      return data;
+       return data;
```

Comment:
ici il y a un espace avant
```suggestion
      return data;
```